docs/development/extensions-core/s3.md

```diff
@@ -64,6 +64,7 @@ In addition to this you need to set additional configuration, specific for [deep
 ### S3 authentication methods
 
 To connect to your S3 bucket (whether deep storage bucket or source bucket), Druid use the following credentials providers chain
+**Note :** *Below authentication chain can be overridden for connecting to source bucket by specifying the [properties](../../ingestion/native-batch.md#s3-input-source) parameter in the ingestionSpec.*
```

Comment:
I think the doc normally uses a carrot to format note-style paragraphs, like the ones on https://druid.apache.org/docs/latest/tutorials/cluster.html#with-zookeeper-on-master for example. 

I'd use that style, or this could just be a regular second sentence in the paragraph. In either case, I'd rephrase a bit: 

> You can override the default credentials provider chain by specifying an access key using [Properties Object parameters](../../ingestion/native-batch.md#s3-input-source) in the ingestionSpec.

Reply:
This is more for like a second sentence in the paragraph. Similar to other note on this page.

Reply:
You can override the default credentials provider chain for connecting to source bucket by specifying an access key and secret key using Properties Object parameters in the ingestionSpec.